src/ssl.c

```diff
@@ -492,6 +492,15 @@ static int f_newcontext(lua_State *L) {
 	return 1;
 }
 
+/* ARG: ctx, hostname
+** RES: true | false */
+static int f_sethostname(lua_State *L) {
+	Context *cfg = luaL_checkudata(L, 1, TYPE_SSL_CONTEXT);
```

Comment:
`Context *cfg`... Umm? It's either `Context *ctx` (your case) or `Config *cfg`.

Reply:
Sorry for the misnaming, fixed in the last commit.